src/main/java/com/minecolonies/coremod/colony/interactionhandling/RecruitmentInteraction.java

```diff
@@ -160,7 +158,7 @@ public void onServerResponseTriggered(final int responseId, final Player player,
                     data.setHomeBuilding(null);
                     data.setJob(null);
 
-                    if (colony.getWorld().random.nextInt(100) <= MineColonies.getConfig().getServer().badVisitorsChance.get())
+                    if (colony.getWorld().random.nextInt(100) <= 2)
```

Comment:
Maybe we could do with a more elaborate RNG here, the larger the colony size the higher chance of bad visitors?
Maybe source this number from raid difficulty too?